src/subtractValue.js

```diff
@@ -0,0 +1,20 @@
+import { flip, subtract } from 'ramda';
+
+/**
+ * Subtracts its first argument from its second argument.
+ *
+ * @func subtractValue
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.22.0|v2.22.0}
+ * @category Math
+ * @sig Number → Number → Number
```

Comment:
Can we please replace with: `Number -> Number -> Number`. Although ramda is using `→` char we use `->` chars, something we can find on the keyboard ;]